tests/drivers/gpio/gpio_basic_api/src/test_gpio_port.c

```diff
@@ -644,7 +644,7 @@ static int pin_get_config(void)
 	zassert_equal(rc, 0, "pin configure failed");
 
 	rc = gpio_pin_get_config(dev, PIN_OUT, &flags_get);
-	if (rc == -ENOTSUP) {
+	if (rc == -ENOSYS) {
```

Comment:
I believe this should not be changed.

E.g. see
https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/drivers/gpio/gpio_get_direction/src/main.c#L43

Reply:
> I believe this should not be changed.
> 
> E.g. see https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/drivers/gpio/gpio_get_direction/src/main.c#L43

@cfriedt, it seems the line you're pointing is not in line with gpio header:
https://github.com/zephyrproject-rtos/zephyr/blob/6cfb18686e1c494d0011aec87bbf24ab530d3a34/include/zephyr/drivers/gpio.h#L881-L882

Reply:
Oh weird - OK - can you make an additional commit to fix the test file that I pointed to? Thanks for pointing that out.